preview/src/index.html

```diff
@@ -4,12 +4,12 @@
 <head>
     <meta charset="utf-8">
     <title>Flow2</title>
-    <link href="https://fonts.googleapis.com/css?family=Roboto:300,400,500" rel="stylesheet">
+    <link type="text/css" rel="stylesheet" href="./fonts.css" />
```

Comment:
`google-fonts-webpack-plugin` loads our Roboto fonts into `fonts.css`. figure we were going to have to do this at some point.